bitnami/mysql/templates/slave-statefulset.yaml

```diff
@@ -195,7 +195,7 @@ spec:
           {{- if or .Values.root.injectSecretsAsVolume }}
           volumeMounts:
             - name: mysql-credentials
-              mountPath: /opt/bitnami/mysql/secrets/
+              mountPath: /opt/bitnami/mysqld-exporter/secrets/
```

Comment:
Modify slave statefulset's  volumeMounts mountPath value  is the same as ENV  parameter MYSQL_ROOT_PASSWORD_FILE